mobile-app/app/screens/AppNavigator/screens/Loans/screens/ConfirmEditLoanSchemeScreenV2.tsx

```diff
@@ -0,0 +1,382 @@
+
+export function ConfirmEditLoanSchemeScreenV2({
+  route,
+  navigation,
+}: Props): JSX.Element {
+  const { vault, loanScheme, fee } = route.params;
+  const { address } = useWalletContext();
+  const addressLabel = useAddressLabel(address);
+
+  // Submit
+  const hasPendingJob = useSelector((state: RootState) =>
+    hasTxQueued(state.transactionQueue)
+  );
+  const hasPendingBroadcastJob = useSelector((state: RootState) =>
+    hasBroadcastQueued(state.ocean)
+  );
+  const dispatch = useAppDispatch();
+  const logger = useLogger();
+  const [isOnPage, setIsOnPage] = useState<boolean>(true);
+
+  function onCancel(): void {
+    navigation.navigate({
+      name: "EditLoanSchemeScreen",
+      params: {
+        vaultId: vault.vaultId,
+      },
+      merge: true,
+    });
+  }
+  async function onSubmit(): Promise<void> {
+    if (hasPendingJob || hasPendingBroadcastJob) {
+      return;
+    }
+    await editLoanScheme(
+      {
+        vaultId: vault.vaultId,
+        loanScheme,
+      },
+      dispatch,
+      () => {
+        onTransactionBroadcast(isOnPage, navigation.dispatch);
+      },
+      () => {},
+      logger
+    );
+  }
+
+  useEffect(() => {
+    setIsOnPage(true);
+    return () => {
+      setIsOnPage(false);
+    };
+  }, []);
+
+  return (
+    <ThemedScrollViewV2 style={tailwind("px-5 py-8")}>
+      <SummaryHeader
+        vaultId={vault.vaultId}
+        addressLabel={addressLabel}
+        address={address}
+      />
+      <SummaryTransactionDetails
+        minColRatio={vault.loanScheme.minColRatio}
+        newMinColRatio={loanScheme.minColRatio}
+        vaultInterest={vault.loanScheme.interestRate}
+        newVaultInterest={loanScheme.interestRate}
+        fee={fee}
+      />
+      <View style={tailwind("my-16")}>
+        <SubmitButtonGroupV2
+          isDisabled={
+            hasPendingJob ||
+            hasPendingBroadcastJob ||
+            vault.loanScheme.id === loanScheme.id
+          }
+          label={translate("screens/ConfirmEditLoanSchemeScreen", "Edit")}
+          onCancel={onCancel}
+          onSubmit={onSubmit}
+          displayCancelBtn
+          title="edit_loan_scheme"
+        />
+      </View>
+    </ThemedScrollViewV2>
+  );
+}
+
+function SummaryHeader(props: {
+  vaultId: string;
+  addressLabel: string | null;
+  address: string | null;
+}): JSX.Element {
+  return (
+    <ThemedViewV2>
+      <ThemedTextV2
+        light={tailwind("text-mono-light-v2-500")}
+        dark={tailwind("text-mono-dark-v2-500")}
+        style={tailwind("text-xs font-normal-v2")}
+        testID="edit_loan_scheme_title"
+      >
+        {translate(
+          "screens/ConfirmEditLoanSchemeScreen",
+          "You are editing loan scheme of"
+        )}
+      </ThemedTextV2>
+      <ThemedTextV2
+        style={tailwind("text-sm font-normal-v2 mt-1")}
+        numberOfLines={1}
+        ellipsizeMode="tail"
+        testID="edit_loan_scheme_vault_id"
+      >
+        {props.vaultId}
```

Comment:
@G9000 Let's use the same format here as the display on previous screen. Concat should be at the middle

<img width="300" alt="image" src="https://user-images.githubusercontent.com/25013382/194835910-5716fc29-8ec9-4e4a-8759-bd538bac2e75.png">